CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff

```diff
@@ -443,18 +463,22 @@
       </trans-unit>
       <trans-unit id="Could not load static JSON" xml:space="preserve">
         <source>Could not load static JSON</source>
+        <target>Could not load static JSON</target>
         <note>Do not translate</note>
       </trans-unit>
       <trans-unit id="Could not parse static JSON" xml:space="preserve">
         <source>Could not parse static JSON</source>
+        <target>Could not parse static JSON</target>
         <note>Do not translate</note>
       </trans-unit>
       <trans-unit id="Credits" xml:space="preserve">
         <source>Credits</source>
+        <target>Sceny w napisach</target>
         <note>Shown as a header on the movie view for the mid- and post-credits scene icons</note>
       </trans-unit>
       <trans-unit id="Credits Lists" xml:space="preserve">
         <source>Credits Lists</source>
+        <target>Listy udziałów filmowych</target>
```

Comment:
Brzmi dość nieporęcznie - na "credits" mówimy po prostu zwykle/popularnie "napisy".

Reply:
Changed to Listy występów